gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/DatastoreTest.java

```diff
@@ -162,8 +151,8 @@ public void setUp() {
 
   @AfterClass
   public static void afterClass() throws IOException, InterruptedException {
-    if (gcdHelper != null) {
-      gcdHelper.stop();
+    if (helper != null) {
```

Comment:
I am not sure this check is necessary (and it is not consistent with beforeClass).